src/ptvsd/__main__.py

```diff
@@ -349,17 +349,17 @@ def escape(ch):
 
 import ptvsd.log
 ptvsd.log.to_file()
-ptvsd.log.info('Debugger successfully injected')
+ptvsd.log.info("Debugger successfully injected")
 
 if ptvsd.options.client:
     from ptvsd._remote import attach
     attach(({host}, {port}))
 else:
     ptvsd.enable_attach()
 '''.format(**locals())
-    print(code)
+    assert "'" not in code
 
-    ptvsd.log.debug('Injecting debugger into target process.')
+    ptvsd.log.debug('Injecting debugger into target process: \n"""{}\n"""'.format(code))
```

Comment:
This needs to be `"... {0} ... "`, because we're using `string.Formatter` rather than `format` for logging, and in Python 2 it doesn't understand unnumbered placeholders like `{}`. Otherwise looks good!

Reply:
Also, can you swap the logging and the assert? This way, if the assert gets tripped, we can see what exactly went wrong in the log - e.g. if one of the formatted values ends up introducing a single quote, we want to know which one did that.

Reply:
Sure, it makes more sense